Apache-Arrow-Flight-Tester/common.py

```diff
@@ -1,91 +1,81 @@
 
-    for batch in response:
-        pprint.pprint(batch.data.to_pydict())
+        return response.schema
 
+    def do_action(self, action_type: str, action_body: bytes) -> list[Result]:
+        """Wrapper around the do_action method of the FlightClient class."""
+        action = flight.Action(action_type, action_body)
+        response = self.flight_client.do_action(action)
 
-def do_put(flight_client: FlightClient, table_name: str, record_batch: pyarrow.RecordBatch) -> None:
-    upload_descriptor = flight.FlightDescriptor.for_path(table_name)
-    writer, _ = flight_client.do_put(upload_descriptor, record_batch.schema)
+        return list(response)
 
-    writer.write(record_batch)
-    writer.close()
+    def list_actions(self) -> list[ActionType]:
+        """Wrapper around the list_actions method of the FlightClient class."""
+        response = self.flight_client.list_actions()
 
+        return list(response)
 
-def do_action(flight_client: FlightClient, action_type: str, action_body: bytes) -> list[Result]:
-    action = flight.Action(action_type, action_body)
-    response = flight_client.do_action(action)
+    def list_table_names(self) -> list[str]:
+        """Return the names of the tables in the server or manager."""
+        flights = self.list_flights()
+        return [table_name.decode("utf-8") for table_name in flights[0].descriptor.path]
 
-    return list(response)
+    def create_table(self, table_name: str, columns: list[tuple[str, str]], model_table=False) -> None:
+        """
+        Create a table in the server or manager with the given name and columns. Each pair in columns should have the
+        format (column_name, column_type).
+        """
+        create_table = "CREATE MODEL TABLE" if model_table else "CREATE TABLE"
+        sql = f"{create_table} {table_name}({', '.join([f'{column[0]} {column[1]}' for column in columns])})"
+        self.do_action("CommandStatementUpdate", str.encode(sql))
 
+    def create_test_tables(self) -> None:
+        """
+        Create a table and a model table using the flight client, print the current tables to ensure the created tables are
+        included, and print the schema for the created table and model table to ensure the tables are created correctly.
+        """
+        print("Creating test tables...")
 
-def list_actions(flight_client: FlightClient) -> list[ActionType]:
-    response = flight_client.list_actions()
+        self.create_table("test_table_1", [("timestamp", "TIMESTAMP"), ("values", "REAL"), ("metadata", "REAL")])
+        self.create_table("test_model_table_1", [
+            ("location", "TAG"), ("install_year", "TAG"), ("model", "TAG"),
+            ("timestamp", "TIMESTAMP"), ("power_output", "FIELD"),
+            ("wind_speed", "FIELD"), ("temperature", "FIELD(5%)")
+        ], model_table=True)
 
-    return list(response)
+        print("\nCurrent tables:")
+        table_names = self.list_table_names()
+        print(table_names)
```

Comment:
Maybe delete this line as the tables names and schemas as printed below.

Reply:
Fixed.